### DISCUSSION TITLE
Computed Columns Through Table Designer

### DISCUSSION THREAD

**Discussion by camvela** [microsoft/azuredatastudio#18911] [Q&A]
Hey All,

I'm very new to SQL so apologies if this is a stupid question. Is it possible to create computed columns via the table designer preview? I've seen many tutorials on SSMS where there is an option for "Computed Column Specification" but I don't seem to see any option like that inside the data studio table designer. Maybe this is too granular for the new feature, but I was hoping there was a way to develop tables with computed columns without writing out all the various lines of code. Thank you so much in advance!

-Cameron

**Comment by alanrenmsft** [comment]
@camvela yes, this is a something we have not yet implemented, here is the issue tracking it: https://github.com/microsoft/azuredatastudio/issues/19117
